ui/src/templates/utils/index.ts

```diff
@@ -58,3 +55,20 @@ export const findVariablesToCreate = (
     v => !existingVariables.find(ev => ev.name === v.attributes.name)
   )
 }
+
+export const hasLabelsRelationships = (resource: {
+  relationships?: Relationships
+}) => !!resource.relationships && !!resource.relationships[TemplateType.Label]
+
+export const getLabelRelationships = (resource: {
+  relationships?: Relationships
+}): LabelRelationship[] => {
+  if (!hasLabelsRelationships(resource)) {
+    return []
+  }
+
+  return [].concat(resource.relationships[TemplateType.Label].data)
```

Comment:
this was just a way to ensure if it's one or more labels it always an array of relationships